### ISSUE TITLE
test compilation failure with aeson >= 1.5.2.0

### ISSUE THREAD

**Issue by bergmark** [Vlix/safe-json#22]
```
    [ 2 of 12] Compiling Instances [Test.QuickCheck.Instances changed]

    /var/stackage/work/unpack-dir/unpacked/safe-json-1.1.0-bed3444b0fc22b82b5c91ce51ee6c0c5bfd1787b701f4463511e75e5693602ea/test/Instances.hs:52:10: error:
        Duplicate instance declarations:
          instance Ord Value -- Defined at test/Instances.hs:52:10
          instance Ord Value
            -- Defined in ‘aeson-1.5.3.0:Data.Aeson.Types.Internal’
       |
    52 | instance Ord Value where
       |          ^^^^^^^^^
```

Will add the tests to expected failures in stackage for now, that can be reverted once this has been fixed (a PR would be appreciated!)

**Comment by Vlix** [comment]
Ah, this is in the `test-suite`... I'll see when I can fix this. Hopefully this weekend.

**Comment by Vlix** [comment]
Published new version `1.1.1` which fixes this issue.
